hail/python/test/hail/test_ir.py

```diff
@@ -46,16 +46,16 @@ def value_irs(self):
         a = ir.Ref('a', env['a'])
         st = ir.Ref('st', env['st'])
         whitenStream = ir.Ref('whitenStream')
-        mat = ir.Ref('mat')
-        aa = ir.Ref('aa', env['aa'])
+        ir.Ref('mat')
+        ir.Ref('aa', env['aa'])
```

Comment:
These shouldn't be side effecting. I'd add their references to the `value_irs` list below to exercise them.